pages/new.js

```diff
@@ -0,0 +1,9 @@
+import Form from "../components/Form/Form";
+
+export default function NewPage({ addCard }) {
+  return (
+    <>
+      <Form onSubmit={addCard} />
+    </>
```

Comment:
```suggestion
      <Form onSubmit={addCard} />
```

Reply:
commonly form pages are called `pages/create.js` but i won't fight you for `pages/new.js` just as a comment :D